apb.yml

```diff
@@ -19,7 +19,13 @@ plans:
   - name: default
     description: Deploy keycloak
     free: True
-    metadata: {}
+    metadata:
+      mobileclient_bind_parameters_data:
```

Comment:
Can we delete this line here https://github.com/aerogearcatalog/keycloak-apb/blob/39b4b6c5732a18ea2020a1161c62ce058082a784/apb.yml#L62 since this value is getting set from the metadata now.

Reply:
That is still needed for bindings that are not created through our custom UI. This only takes effect through our integrations UI.